coffee-module/coffee-module-mp/coffee-module-mp-restclient/src/main/java/hu/icellmobilsoft/coffee/module/mp/restclient/provider/DefaultLoggerClientRequestFilter.java

```diff
@@ -64,8 +72,39 @@ public void filter(ClientRequestContext requestContext) throws IOException {
         msg.append(">> ").append(getClass().getName()).append(" request ->\n");
         msg.append(logUrl(requestContext));
         msg.append(logHeader(requestContext));
-        msg.append(logEntity(requestContext));
-        LogProducer.logToAppLogger((AppLogger appLogger) -> appLogger.info(msg.toString()), DefaultLoggerClientRequestFilter.class);
+        logContainer.setLogBuilder(msg);
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void aroundWriteTo(WriterInterceptorContext context) throws IOException, WebApplicationException {
+        int maxResponseEntityLogSize = RestLoggerUtil.getMaxEntityLogSize(context, LogSpecifierTarget.CLIENT_REQUEST);
+        if (maxResponseEntityLogSize == LogSpecifier.NO_LOG) {
+            context.proceed();
+            return;
+        }
+
+        OutputStream originalStream = context.getOutputStream();
+        byte[] entityCopy = new byte[0];
+        OutputStreamCopier osc = new OutputStreamCopier(originalStream);
```

Comment:
Felesleges a teljes `originalStream`-et a memóriába másolni, ha csak `maxResponseEntityLogSize` méretnyit fogunk printelni a `requestResponseLogger.printEntity()` híváskor. Az `OutputStreamCopier` konstruktorban kaphatna egy limit-et (jelen esetben `maxResponseEntityLogSize`-nyit), hogy mennyi byte-ot másoljon.

Reply:
lezarom, adok fel egy ujat, azota mar több problema is bejött, köszi az eszrevételeket.